service/replication.go

```diff
@@ -137,10 +145,14 @@ func (s *service) CreateRemoteVolume(ctx context.Context,
 		quota, err := remoteIsiConfig.isiSvc.client.GetQuotaWithPath(ctx, exportPath)
 		log.Info("Get quota", quota)
 		if err != nil {
-			log.Info("Remote quota doesn't exist, create it")
-			quotaID, err = remoteIsiConfig.isiSvc.CreateQuota(ctx, exportPath, volName, volumeSize, s.opts.QuotaEnabled)
-			if err != nil {
-				return nil, status.Errorf(codes.Internal, "can't create volume quota %s", err.Error())
+			if strings.Contains(err.Error(), "not found:") {
+				log.Info("Remote quota doesn't exist, create it")
```

Comment:
Change 'create' to 'creating' to better describe what is happening. 'create it' sounds like the user should go and perform the action manually.

Reply:
Will correct the log texts in the next PR. There will be one soon for the improvements I have planned.